xml/System.Security.Cryptography.Pkcs/SignerInfo.xml

```diff
@@ -59,9 +59,21 @@
         <Parameter Name="asnEncodedData" Type="System.Security.Cryptography.AsnEncodedData" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="asnEncodedData">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="asnEncodedData">The attribute to add, an object with Abstract Syntax Notation One (ASN.1)-encoded data.</param>
+        <summary>Adds the specified attribute to the current instance's attribute collection. </summary>
+        <remarks>  
+          <format type="text/markdown"><![CDATA[  
+
+If the attribute already exists, it is merged with the specified attribute. 
```

Comment:
`with the existing attribute.`?

Reply:
@krwq, the code has a comment stating that if the attribute already exists, the specified one will get merged with the existing one:

https://github.com/dotnet/corefx/pull/29408/files#diff-c6915c2b9c9dad1198786b7a8bd5cd9dR159

Is that not accurate?

Reply:
it is, but the xml added says "specified attribute" and not "existing attribute"

Reply:
If the attribute already exists -> meaning, the one that is already in the attribute collection.

it (the existing attribute) is merged with the specified attribute -> The specified attribute is the one passed as an argument

Reply:
We've been using "specified" to refer to the passed parameter. But if you feel like we should use different language, please provide a suggestion.

Reply:
Ok I think I understand what you mean. I'll change it to make it clearer:

```suggestion
If the specified attribute already exists in the document, it is merged with the existing one.
```